recipe/meta.yaml

```diff
@@ -21,12 +21,12 @@ requirements:
     - ninja      # [win]
 
   host:
+    - python
     - boost
     - eigen
     - obake-devel
     - mppp
     - audi {{ version }}
-    - python
     #- pybind11
```

Comment:
you will need pybind here, dont you?

Reply:
I am installing pybind11 HEAD when building the recipe as the currently released pybind has problems when compiled with the c++17 standard.

Later, when the new pybnd11 will be released and available in conda I can remove such a work around.